scripts/migrate-adapter.sh

```diff
@@ -1,17 +1,17 @@
-repo=$1
-source_branch=$2
-target_branch=$3
+ADAPTER=$1
+SOURCE_BRANCH=$2
+TARGET_BRANCH=$3
 
 # create a remote for the legacy adapter repo and fetch the latest commits
-git remote remove old || true
-git remote add old https://github.com/dbt-labs/$repo.git
-git fetch old
+git remote remove adapter || true
```

Comment:
Should the remote name be `$ADAPTER`? What if I want to port PRs from multiple adapter repos?

Reply:
I figured we would be doing this one at a time (especially if we're migrating the entire adapter). So we'd wind up adding/removing each time. I could update that though if you feel strongly about it.